docs/jep/9407-type-promotion.ipynb

```diff
@@ -296,7 +296,7 @@
JSIWHvK4tbUV8+fPx6JFi6ht9QB1dXUYMmQIoqKiYDaboaoqAoEA2tra4Pf7Adx9xUNWVhZKSkp0H3tRzmHCFzxBtFtleHXXCjt//ny3T4XJsoyVK1eioKCgPVY2mw27du3C888/32lco+ywTcTGQx6rqoqBAwc+1EqtUXm9XhQVFSEqKgoDBgy4509kZGT7Z3mIvSjnMMMUPAC9j0Rv7tcKO3v2LG7evNnp81FRUVixYgU2bdoEk8kk7DvUiLFQHhsXxT60DFXwBCmKgsbGRtTX1/d5xSe4ohMfHw+n08n9Up9e1NfX4/PPP8f3vvc9KIpyz99brVZ8+9vfxt69e2E2m/HVV1/16Y3BoSZJEoYNG4bo6OhwT4XoTHNzM+WxQVHsQ8sc7gmEQ7BQiYuLg8fjgdvthsfjgc/nA4BuK9hgUWS1WmG32+FwOGC327mudvUoPj4eiYmJsNls9zzOCdx9bcZnn32GYcOG4dKlS8xPFPv27cOuXbtQVVUFu92O1NRU5OTkIC0trV/jqqoKt9vN9cmCaMPtdjPL41mzZqGhoQGyLMNsNmP8+PFYvXo1szYU5TFbrGIfjLvJZILJZEJKSgrmzp2LF154gVnXQYTYG7LgCZIkCdHR0e0BVFUVfr8ffr8fgUAAqqpCkiTIsgyLxQKLxUIFTggcO3as0+qO2WyGzWaD1+tFWloaZs+ejczMzG4f9+yP9957Dzt27MBvfvMbZGVlISIiAv/5z39QWlra74IHAPP5EjGwzouCggJkZmbC5/MhPz8fGzduxJYtW5iNT3l8f1evXsXcuXOxatUqfP/737/v6j/L7zIYd7fbjZMnT+KNN97AmTNnkJ+fz+wYvMfe0AVPV8G9c2gvifA6fPgwWltbERERgdGjR2Pu3LmYMWMGMjIyYLFYANwtTi9cuMDsmG63G4WFhVi/fj2mT5/e/vPs7GxkZ2czOUZwTygqmkmQqqrtK8usWa1WzJgxA2+++SbTcSmP76+urg7nz59HTk4OfvWrX2HTpk3dFj5axd7hcODJJ59EfHw8Fi1ahCVLlmDEiBFMxuY99nSHLdGdvLw87Nu3D01NTTh9+jTWrVuHJ554or3YAdD+aCcr5eXl8Pv9mDZtGtNxu2I9b8I3LfOhpaUFBw4cwLhx45iPTXl8f1artf3J4JycHCQnJ+ODDz7otHKt9Xf42GOPYfDgwSgrK2M6Ls+xpxUeojvp6ekP/Izf74ckScwe5WxqakJMTAzMZu1+JSRJgt/vpxVE0o51HgPAypUrYTab0dLSAqfTiW3btjEbG6A8flgejwcej+eeFR8tYt/VoEGDcPv2bWbj8R57KngIl1jfrBwTE4Ompia0tbVpWvTw8DQGCR0t8mHz5s3IzMyEoigoLS3F0qVLsWfPHmZ7qLjdbuTk5ODIkSNMxhON3+/v9oGLYOGzaNEirFmzBidPntR8Ljdu3MCAAQOYjsnzOYwKHsIl1ldFjz/+OCwWC0pKSjBz5kymY3dkwF0gyH1omQ8mkwnTp0/HunXrUFZWxiyvo6OjsX79ekRERDAZTzTnzp3Diy++iObm5k4/j4iIgCzLmDt3LtatW6f5ueDcuXOoq6vDt771Labj8nwOo4KHcIn1TXMOhwMrVqzAhg0bYDKZkJWVBbPZjKNHj+LEiRN45ZVXmByH15v9iDa0zAdVVVFaWoo7d+4gJSWF2biSJGHgwIHMVw5EcevWrU6PgkdERMBkMmHhwoVYu3YtEhMTAYBpq6mj5uZmnDp1Cps2bcIzzzyDkSNHMh2f53MYFTyES1rsaL1kyRLExcXhnXfeweuvvw6bzYbRo0cjJyeH2TFoJ27SkRb5kJubC1mWIUkSEhISkJ+fj+HDhzM9BuXx/SmK0mOhE8T6O8zNzYXJZIIsy0hJScFLL72EBQsWMD0GwHfsDbnTMuGfz+dDZWUlV8urkiRh+PDh3N7wR9ijPBZPRUUFxo4di8WLF3db6ARR7EOPVngIlzo+os4TXudNtMFrPvA671BITU3FnTt3EBUVdd/P8fod8jpvgPbhIZwKbhLJE6vVynX/m7BHeSymBxU7AMU+HKjgIdyy2+3hnsJD4W2+JDR4ywve5qtnvH2XvM23Kyp4CLccDgc3N9BJkgSHwxHuaRAdojw2Lop9aPHxTRPSDbvdzs3JwmQycX91RLRBeWxcFPvQ4uObJqQbkiQhLi5O9z1lSZIQHx+v+3mS8KA8Ni6KfWhRwUO4FhsbG+4p9IrT6Qz3FIiOUR4bF8U+dKjgIVwzmUxwOp26vfKQJAlOpxMmkyncUyE6RnlsXBT70KGCh3DP5XLp9pfRZDLB5XKFexqEA5THxkWxDw0qeAj3ZFlGYmKi7q6QJElCYmIiNzclkvCiPDYuin1oiPFvQQzPbrfralk4uAzM+1MNJLQoj42LYq89KniIMFwuFyIjI8N+wpAkCZGRkcIsA5PQojw2Loq9tqjgIcKQZRlJSUlh3f48uF18UlKSMMvAJLQoj42LYq8tels6EY6iKKiurobX6w3pm4iDV0VJSUm6vQGR8IPy2Lgo9tqggocIKRAIoLa2Fo2NjSE5YQT73S6XS7irIhI+lMfGRbFnjwoeIjSPx4OamhooiqLJSUOSJJhMJiQmJgp1cx/RF8pj46LYs0MFDxFexyslAExOGsH+uuhXREQ/KI+Ni2LPBhU8xDAURUFjYyPq6+v7fLUUvBqKj48XZvdRwhfKY+Oi2PcPFTzEcFRVhcfjgdvthsfjgc/nA4Bun4oI/npYrVbY7XY4HA7Y7fawPzZKCOWxcVHs+4YKHmJ4qqrC7/fD7/cjEAhAVVVIkgRZlmGxWGCxWAx5ciB8oTw2Lop971DBQwghhBDhiX+XEiGEEEIMjwoeQgghhAiPCh5CCCGECI8KHkIIIYQIjwoeQgghhAiPCh5CCCGECI8KHkIIIYQIjwoeQgghhAiPCh5CCCGECO//AcN1Ixa6KgO0AAAAAElFTkSuQmCC\n",
```

Comment:
What changes did you make to these images? It's difficult to tell from the code diff. Perhaps revert, since this doesn't have anything to do with fixing typos?